torch/distributed/rpc/__init__.py

```diff
@@ -156,7 +156,10 @@ def init_rpc(
             logger.warning(
                 "RPC was initialized with the PROCESS_GROUP backend which is "
                 "deprecated and slated to be removed and superseded by the TENSORPIPE "
-                "backend. It is recommended to migrate to the TENSORPIPE backend."
+                "backend. It is recommended to migrate to the TENSORPIPE backend. "
+                "PyTorch v1.9 will be the last release that carries PROCESS_GROUP "
+                "RPC backend. If you have concerns or suggestions please comment in "
+                "https://github.com/pytorch/pytorch/issues/55615"
```

Comment:
Just noticed this is a `logger.warning`, shouldn't we use python warnings instead: `warnings.warn(..., DeprecationWarning)`? This seems like the standard practice for deprecation warnings.

Reply:
good point, let me update.